packages/workflow/src/documents.ts

```diff
@@ -58,11 +62,11 @@ export async function uploadCertificateAttachmentsToDocumentsStore<
     certificateDetails.collector.affidavit
   ) {
     for (const affidavit of certificateDetails.collector.affidavit) {
-      affidavit.data = await uploadBase64ToMinio(affidavit.data, authHeader)
+      affidavit.data = await uploadSVGToMinio(affidavit.data, authHeader)
```

Comment:
@euanmillar How do we know that the uploaded affidavit will always be an SVG?

Reply:
Thanks @Zangetsu101 for the catch.  Changed affidavit appropriately